src/stylesheets/core/_variables.scss

```diff
@@ -143,6 +143,7 @@ $box-shadow:              0 0 2px $color-shadow !default;
 $focus-outline:           2px dotted $color-gray-light;
 $focus-spacing:           3px;
 $nav-width:               951px !default;
+$current-highlight-border-width: 0.4rem !default; // must be in rem for math
```

Comment:
@thisisdano for this new variable, a few things....
- Each variable section's values should align, here it extends past the other values.
- Is the variable applicable to more things besides the sidenav? 
- Would it work to call it `$current-border-width` or do we need `highlight`?
- I think we can remove the comment "// must be in rem for math" bc many other variables are in rem and we don't state this